src/pages/back-to-office/back-to-office.scss

```diff
@@ -0,0 +1,26 @@
+page-back-to-office {
```

Comment:
Feels like this the CSS for this page could be made a bit more concise with flexbox.